teacher.rb

```diff
@@ -1,4 +1,16 @@
+require_relative 'highfive'
+require_relative 'setphase'
+require_relative 'salary'
+require_relative 'receiveraise'
+
+
 class Teacher
+  include HighFive
+  include SetPhase
+  include Salary
+  include ReceiveRaise
+
+
   attr_reader :age, :salary, :phase, :performance_rating, :target_raise
```

Comment:
`Teacher` has a `performance_rating`, but `ApprenticeTeacher` does not.  Having `ApprenticeTeacher` inherit from `Teacher` gives `performance_rating` to `ApprenticeTeacher` as well.  Instead, have them both inherit from a common superclass, maybe called `Educator` or `Staff`